commons/src/main/java/org/eclipse/kapua/commons/event/jms/JMSServiceEventBus.java

```diff
@@ -156,9 +158,8 @@ public ServiceEventBus getEventBus() {
     }
 
     private void waitBeforeRetry() {
-        // wait a bit
         try {
-            Thread.sleep(2000);// TODO move to configuration
+            Thread.sleep(WAIT_BETWEEN_RECONNECTION_ATTEMPT);
```

Comment:
Should we add a backoff function here, instead of a fixed sleep?

Reply:
I preferer to stay with a fixed (and not too big) amount of waiting time since I'm expecting that the service broker will be down for not too long periods. With a backof the risk is to have the waiting time very huge at a certain time

Reply:
With a backoff we could define a maximum value, but I'm okay proceeding with this simpler solution unless we find a need to change it otherwise.